src/lib/y2storage/encryption.rb

```diff
@@ -440,6 +468,19 @@ def mount_point_to_dm_name
       mount_point.path.gsub(/^\/|\/$/, "").gsub("/", "_")
     end
 
+    # Checks whether the initial value of mount_by was forced to DEVICE by libstorage-ng due to lack
+    # of information or whether is the result of a proper initialization during normal operation
+    #
+    # @see #adjust_mount_by
+    #
+    # @return [Boolean]
+    def probed_without_crypttab?
+      return false unless exists_in_probed?
+      return !in_etc_crypttab? if in_etc_initial.nil?
```

Comment:
What happens when `#assing_etc_attribute` is called before setting `in_etc_initial`?